src/components/editor/Controls.tsx

```diff
@@ -12,37 +13,70 @@ const Label = ({ text, children }: {
 	</label>
 }
 
-export default function Controls({ exportHandler, exporting }: {
-	exportHandler: (
-		data: {
-			fps: number,
-			size: number,
-			speed: number,
-			bounce: boolean,
-			loop_gif: boolean
-		}) => void,
-	exporting: boolean
+const getEstimatedFileSize = (
+	fps: number,
+	width: number,
+	height: number,
+	quality: number,
+	durationInFrames: number
+) => {
+	const durationInSeconds = durationInFrames / CONSTANTS.previewFps;
+	const totalFrames = durationInSeconds * fps;
+	const qFactor = quality / 100;
+
+	const totalPixels = (width * height * totalFrames) / 3;
```

Comment:
Why do we divide by 3 here?

Reply:
This even I didn't understand yet. The Giksi app version divides by 3 so I've bought that over here. Claude/GhatGPT just suggested changing that `/` to multiplying but of course that yielded very far off results 🤷‍♂️